### DISCUSSION TITLE
New Experiment

### DISCUSSION THREAD

**Discussion by StoppedwummSites** [Our-Code-24/opensearch#54] [Announcements]
We worked hard and now we are proud to present:
# Rewards
We are using auth0 to manage accounts!

Soon, you will be able to:

- Log in
- Collect points

More account features will come as experiments.

Share your opinion with us in the comments!

Best Regards,
Stoppedwumm

**Comment by Stoppedwumm** [comment]
In a few days, a new pr will be opened so you can test it, we currently are developing methods so the keys are not revealed.
